docs/LogGuide.md

```diff
@@ -1122,6 +1140,25 @@ You cannot count on the time remaining to be precise.
 In rare cases, the time will already have counted down a tiny bit.
 This matters for cases such as ucob Nael phase doom debuffs.
 
+The 'stacks' value may also indicate other information about the buff. For example, Mudra will show different stack
+values for different combinations of Mudra. The only way to be sure that you are getting a "real" stack value is by
+cross-referencing with game data (such as via Xivapi) to see what the game says the max stack amount for that ability
+is (0 is the max for most abilities).
```

Comment:
The actual use for the field is more generic, it can be used for stacks but other purposes too. It's on a per-status case tho. Side note, `xivapi` is just `xivapi`, all lowercase.